pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -1050,3 +1079,12 @@ def test_load_non_canonical_image_table():
         result.sort_index(axis="columns").drop("Metadata_Site_Count", axis="columns"),
         sc_aggregated_df,
     )
+
+def test_singlecells_default_datatype():
```

Comment:
this can probably be deleted, right?

Reply:
Thank you, yes, apologies. I'll remove this block.